app/frontend/src/assets/scss/style.scss

```diff
@@ -443,6 +443,24 @@ a,
     @extend :disabled;
     background-color: rgba(0, 0, 0, 0.12);
   }
+  // the drag/drop area needs to line up with vuetify v-application ul styles...
+  .formio-component-file .fileSelector {
+    margin-left: 24px;
+  }
+}
+
+.form-wrapper {
+   // the drag/drop area needs to line up with vuetify v-application ul styles...
+  .formio-component-file {
+    .fileSelector {
+      margin-left: 24px;
+    }
+    .file {
+      .row {
+        margin-left: 24px;
```

Comment:
If possible, would be better to define this in rem units instead of px (I think this would be 1.5rem).

Reply:
only in place because we are making this match exactly what vuetify does to the ul under the file selector - and it uses 24px.